simulator/src/main/java/com/hazelcast/simulator/worker/TestContainer.java

```diff
@@ -152,10 +152,14 @@ public T getTestContext() {
         return testContext;
     }
 
-    public long getOperationCount() throws Exception {
-        Long count = invokeMethod((operationCountWorkerInstance != null) ? operationCountWorkerInstance : testClassInstance,
-                operationCountMethod);
-        return (count == null ? -1 : count);
+    public long getOperationCount() {
+        try {
+            Long count = invokeMethod((operationCountWorkerInstance != null) ? operationCountWorkerInstance : testClassInstance,
+                    operationCountMethod);
+            return (count == null ? -1 : count);
+        } catch (Exception ignored) {
+            return -1;
```

Comment:
it would be good to have `log.finest(...)` - as swallowing exceptions leads to a nightmare troubleshooting when things go wrong.